### DISCUSSION TITLE
Secondary development of drawio

### DISCUSSION THREAD

**Discussion by jayyliang** [jgraph/drawio#4355] [General]
Can I develop a commercial product based on drawio? If possible, what do I need to pay attention to in terms of copyright or compliance?

Looking forward to your answer

**Comment by davidjgraph** [comment]
You can use the source code in this repo under the Apache 2.0, which does allow you to create a commercial product. You cannot use our trademarks, like the "draw.io" or "diagrams.net" names, or our logos.

You cannot use any source code that we do not put in a repo here on github.

You must host all the software yourself, our online version forbids you directly using anything hosted by us.

The JGraph provided icons and diagram templates are licensed under the [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/), if you use those you must attribute this project in your software.

Apart from that, I wish you good luck in your project :).
